### DISCUSSION TITLE
How to cache the popup edit content

### DISCUSSION THREAD

**Discussion by JianYiheng** [quasarframework/quasar#12317] [General - Components / Directives / etc] [answered]
I am writing to ask how to cache the popup edit content.
[https://codepen.io/jianyiheng/pen/gOXMgWW](url)

![image](https://user-images.githubusercontent.com/20198891/152266281-cf67eca3-54e3-44b4-9dd9-a0f372436c75.png)

What I want is store the content when mouse click outside the text-area.

![image](https://user-images.githubusercontent.com/20198891/152269799-00cd17cb-13e1-48c5-9868-c66d3fd86a28.png)

As shown in above picture, the initial content is "demo title". After I type "new content" and click outside the input area, a function should be triggered to save the modified content to sessionStorage. How should I implement this function?

I have tried:
1. @save: This event is triggered only when I press "enter". I want it be triggered when I click outside the editor.
2. @hide: I want to cache only content is updated. But, "hide" is triggered whenever the popup edit is hiden.

Thank in advance.

**Comment by MilosPaunovic** [comment, marked as answer]
You could use `@update:model-value` event of your `QInput` to detect value change and call the method to save data to SessionStorage.
